### DISCUSSION TITLE
Extended dateClick function

### DISCUSSION THREAD

**Discussion by stwebage** [vkurko/calendar#164] [Q&A] [answered]
Good afternoon, I wanted to clarify whether it is possible to somehow expand the functionality of dateClick, at the moment you can get only the resourceID and the date on which we clicked directly from it. And how to make it so that, along with the date, it would be possible to display the time interval, the ability to immediately create an event in the time interval.

**Comment by vkurko** [comment, marked as answer]
Hello. Have you looked at the [selectable](https://github.com/vkurko/calendar#selectable) and [select](https://github.com/vkurko/calendar#select) options?

**Comment by stwebage** [comment]
Thank you very much for the prompt response, this is exactly what I needed. Tor one small question, but is there timeGridMonth and not dayGridMonth?

**Comment by vkurko** [comment]
Take a look at the [duration](https://github.com/vkurko/calendar#duration) option. But I doubt that in the `timeGrid` mode with a `duration` of 1 month, the calendar can be used normally. Some kind of horizontal scroll is needed.

**Reply by stwebage** [reply]
Thanks for the answer
